apisix/plugins/limit-count/limit-count-redis.lua

```diff
@@ -72,7 +72,11 @@ function _M.incoming(self, key)
     key = self.plugin_name .. tostring(key)
 
     -- todo: test case
-    local ret = red:ttl(key) or -2
+    local ret, err = red:ttl(key)
+    if not ret then
+        return false, "failed to get redis ttl: " .. err
```

Comment:
we'd better contain the `key` in the error message, it is useful

Reply:
OK, good idea.